### PR TITLE
Elasticsearch populate

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: en/developer_guide/elasticsearch_add_new_indexed_document.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+``ElasticaSchemaInitializer``.
+
+This initializer should be tagged with :
+
+.. code-block:: yaml
+
+    tags:
+        - { name: open_orchestra_elastica.schema_initializer.strategy }
+
+This initializer will be called by the console command ``orchestra:elastica:schema:create``.
```
Comment (ngilain) [inline en/developer_guide/elasticsearch_add_new_indexed_document.rst]: What happens if several services are tagged with open_orchestra_elastica.schema_initializer.strategy ? What is the criteria to choose one or another strategy ?

File: en/developer_guide/elasticsearch_add_new_indexed_document.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+``ElasticaPopulatorInterface``.
+
+This populator should be tagged with :
+
+.. code-block:: yaml
+
+    tags:
+        - { name: open_orchestra_elastica.populator.strategy}
+
+This populator will query all the documents that you want to index from the database. This
```
Comment (ngilain) [inline en/developer_guide/elasticsearch_add_new_indexed_document.rst]: Same remark here, what is the criteria for the manager to choose one or another strategy ?

File: en/developer_guide/elasticsearch_add_new_indexed_document.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+The listener should call an indexor that will implement ``DocumentIndexorInterface``
+
+The indexor will get the document and index an elastica document. The indexor will call
+the transformer that you created in the previous section to perform the transformation.
+
+Delete a document
+'''''''''''''''''
+
+When a document is deleted, you should listen tto the ``DocumentEvents::DELETE`` event.
```
Comment (ngilain) [inline en/developer_guide/elasticsearch_add_new_indexed_document.rst]: listen to

File: en/developer_guide/elasticsearch_indexation.rst [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+Add the `Elasticsearch`_ listening address in the ``app/config/config.yml`` file :
+
+.. code-block:: yaml
+
+    open_orchestra_elastica:
+        host: 192.168.33.10
+
+``192.168.33.10`` is our local configuration, this can vary on your installation.
+
+After running the ``composer update`` command, you would see some new command appear when running
```
Comment (ngilain) [inline en/developer_guide/elasticsearch_indexation.rst]: you should see

Comment (nicolasThal) [inline en/developer_guide/elasticsearch_indexation.rst]: fix in the other pr

**Comment by ngilain** [comment]
:+1:

**Comment by veolys-nanne** [comment]
:+1:

**Pull request merged**